core-kotlin-modules/core-kotlin-collections/src/main/kotlin/com/baeldung/kotlin/collections/AggregateOperations.kt

```diff
@@ -0,0 +1,107 @@
+package com.baeldung.kotlin.collections
```

Comment:
Let's make the package more specific, e.g. com.baeldung.kotlin.collections.aggregate

Reply:
@KevinGilmore Moved it to a new module.